### PR TITLE
feat: add get all codes lists

### PR DESCRIPTION
**Pull request by EmmanuelDemey**

### CODE REVIEW

File: src/main/java/fr/insee/rmes/webservice/CodeListsResources.java [hunk 1]
```diff
@@ -44,6 +44,21 @@
 	CodeListService codeListService;
 
 
+	@GET
+	@Path("/")
+	@Produces(MediaType.APPLICATION_JSON)
+	@Operation(operationId = "getAllCodesLists", summary = "List of codes",
+			responses = { @ApiResponse(content = @Content(mediaType = "application/json", schema = @Schema(implementation = CodeList.class)))})
```
Comment (alicela) [inline src/main/java/fr/insee/rmes/webservice/CodeListsResources.java:51]: type Array 

schema=@Schema(type="array",implementation=

File: src/main/resources/request/codes-list/getAllCodesLists.ftlh [hunk 2]
```diff
@@ -0,0 +1,18 @@
+SELECT ?value ?label ?range
+WHERE {
+{
+GRAPH <http://rdf.insee.fr/graphes/codes> {
```
Comment (alicela) [inline src/main/resources/request/codes-list/getAllCodesLists.ftlh:4]: use Config.CODELIST_GRAPH as a param of the query (in the hashmap)

File: src/main/resources/request/codes-list/getAllCodesLists.ftlh [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+?value rdf:type skos:ConceptScheme .
```
Comment (alicela) [inline src/main/resources/request/codes-list/getAllCodesLists.ftlh:5]: Rename value => uri

File: src/main/resources/request/codes-list/getAllCodesLists.ftlh [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+?value skos:prefLabel ?label .
+FILTER(lang(?label) = 'fr') .
+?value rdfs:seeAlso ?range
```
Comment (alicela) [inline src/main/resources/request/codes-list/getAllCodesLists.ftlh:8]: i've to check with florian but i think that it's ?range rdfs:seeAlso ?value

File: src/main/resources/request/codes-list/getAllCodesLists.ftlh [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+}
+}
+UNION
+{
+?value ?b xkos:ClassificationLevel .
```
Comment (alicela) [inline src/main/resources/request/codes-list/getAllCodesLists.ftlh:13]: ?b = rdf:type = a ?

Comment (EmmanuelDemey) [reply src/main/resources/request/codes-list/getAllCodesLists.ftlh:13]: I do not understand this comment.

File: src/main/resources/request/codes-list/getAllCodesLists.ftlh (hunk 4 above)
Comment (EmmanuelDemey) [reply src/main/resources/request/codes-list/getAllCodesLists.ftlh:8]: I have just sent an email to Florian.

Comment (EmmanuelDemey) [reply src/main/resources/request/codes-list/getAllCodesLists.ftlh:8]: Florian just said you are right :D

**Pull request merged**